### ISSUE TITLE
Incorrect numeric filters

### ISSUE THREAD

**Issue by renancadore** [DataTables/SearchBuilder#12]
Hello, I'm having some difficulties with searchBuilder's numeric filters. 

If I have, for example, in the Amount column, the value -200.000,50, and configure a filter less than -20000000, the line continues to be displayed in the dataTable. 

The same behavior can be seen with positive values with the greater than filter.

Here I have an example gist of how to simulate the behavior and evidences of the error:

https://gist.github.com/renancadore/1c3203a575ea2d01fbbee147ec02c967

Without Filters:
![image](https://github.com/DataTables/SearchBuilder/assets/87496015/e26bf3a7-16aa-44fe-96bd-849744243b96)

Greater than:
![image](https://github.com/DataTables/SearchBuilder/assets/87496015/a7e6a921-5374-4906-bf67-3f064e420ead)

Less than:
![image](https://github.com/DataTables/SearchBuilder/assets/87496015/121ca017-74ce-41db-b6d9-51e47a8dad71)

Someone can help me with this?

**Comment by AllanJard** [comment]
Running test case here: https://live.datatables.net/jinodowu/1/edit (thanks for the gist).

It appears to be something to do with the comma decimal. If I comment out the language url, then it works as expected.

**Comment by AllanJard** [comment]
[Fix committed here](https://github.com/DataTables/SearchBuilder/commit/37be65765a8778b1213c54e5a28790bdd9ea1355). I should note that this is on a branch of SearchBuilder for which I'm currently doing the development to make it work with DataTables 2, which should be out in the next month or so.

The fix could readily be backported to a static file for SearchBuilder if you need it, or you could potentially hang on until it gets merged to master.

Thanks for reporting this error!

**Comment by renancadore** [comment]
Hi @AllanJard, thanks for the fix! 

I've replaced a static file with yours changes and that's worked in the conditions that I have exemplified.

But I think we have another problem here.

When I input a value with a decimal comma in the filter, the value is already parsed when it is passed to parseNumber function. i.g. 200000,49 will be parsed to 20000049. 

Here's the evidences:
![image](https://github.com/DataTables/SearchBuilder/assets/87496015/77aba5ed-3e47-43da-8ea1-da8ce49f87fe)

![image](https://github.com/DataTables/SearchBuilder/assets/87496015/faad7de0-b956-456d-b020-06cd2af97934)
